templates/bet/Inspector.tsx

```
@@ -0,0 +1,257 @@
+                          username: arbitratorInputRef.current.value,
+                          fid: arbitratorData.fid,
+                          custody_address: arbitratorData.custody_address,
+                          pfp_url: arbitratorData.pfp_url,
+                      }
+                    : null,
+            })
+
+            if (opponentInputRef.current) opponentInputRef.current.value = ''
+            if (arbitratorInputRef.current) arbitratorInputRef.current.value = ''
+        } catch (error) {
+            throw new FrameError(`Invalid opponent or arbitrator farcaster username`)
```

Biome

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)